### ISSUE TITLE
Don't add Mootools or debug.css in PageRegular if the 'hideDebugBar' config setting is set to true

### ISSUE THREAD

**Issue by adamffyl** [contao/core#8307]
This is in PageRegular.php on lines 427 and 642.  Please let me know if you need more clarification.  Thanks!

**Comment by Aybee** [comment]
:+1:

**Comment by leofeyer** [comment]
@contao/developers Is this a bug or a feature? Possible side-effects?

**Comment by Toflar** [comment]
Imho that's a bug and I don't see any side effects.

**Comment by aschempp** [comment]
What is `hideDebugBar`? :D

**Comment by Aybee** [comment]
> What is hideDebugBar? :laughing:

lol, I also discovered it by this ticket right now.

**Comment by adamffyl** [comment]
I think it's a bug because I was working with a jQuery plugin and Mootools was conflicting with it, so I purposely excluded Mootools for that reason, only to find that Mootools was being added by PageRegular anyways.

**Comment by leofeyer** [comment]
> What is `hideDebugBar`? 😆 

Something @discordier wanted to have to disable the debug bar in his debug extension.

**Comment by leofeyer** [comment]
> I think it's a bug because I was working with a jQuery plugin and Mootools was conflicting with it, so I purposely excluded Mootools for that reason

@adamffyl That's now a bug. Just run jQuery in a separate scope, then it will no longer conflict with anything.

```js
<script>
jQuery(function($) {
    // your code goes here
});
</script>
```

**Comment by adamffyl** [comment]
Thanks @leofeyer.  I actually was doing that, but I think the root of the problem was really because the Mootools "Event" class was trying to execute an event callback from a jQuery script (Slick carousel http://bit.ly/1jqBdSk).  Unfortunately, I didn't have time to track it down to the actual line of code, but removing Mootools did the trick.    =)

**Comment by leofeyer** [comment]
Fixed in c6e0cba3063300403260bcf0644370f0d861a2d9 then.
